mindsdb_sql/parser/dialects/mindsdb/skills.py

```diff
@@ -0,0 +1,93 @@
+from mindsdb_sql.parser.ast.base import ASTNode
+from mindsdb_sql.parser.utils import indent
+
+
+class CreateSkill(ASTNode):
+    """
+    Node for creating a new skill
+    """
+
+    def __init__(self, name, type, params, if_not_exists=False, *args, **kwargs):
+        """
+        Parameters:
+            name (Identifier): name of the skill to create
+            type (str): type of the skill to create
+            params (dict): USING parameters to create the skill with
+            if_not_exists (bool): if True, do not raise an error if the skill exists 
+        """
+        super().__init__(*args, **kwargs)
+        self.name = name
+        self.type = type
+        self.params = params
+        self.if_not_exists = if_not_exists
+
+    def to_tree(self, level=0, *args, **kwargs):
+        ind = indent(level)
+        out_str = f'{ind}CreateSkill(' \
+                  f'if_not_exists={self.if_not_exists}' \
+                  f'name={self.name.to_string()}, ' \
+                  f'type={self.type}, ' \
+                  f'params={self.params})'
+        return out_str
+
+    def get_string(self, *args, **kwargs):
+        using_ar = [f'{k}={repr(v)}' for k, v in self.params.items()]
+        using_str = ', '.join(using_ar)
+
+        out_str = f'CREATE SKILL {"IF NOT EXISTS" if self.if_not_exists else ""}{self.name.to_string()} USING {using_str}'
```

Comment:
two potential problems here:
- 'exists' word can be joined with name (no space between them)
- type is not involved here